source/core/TDCodecVia.cpp

```diff
@@ -2329,6 +2329,56 @@ void TDViaFormatter::FormatClusterData(TypeRef type, void *pData)
     _string->Append(Fmt()._clusterPost);
 }
 //------------------------------------------------------------
+void TDViaFormatter::FormatVariant(TypeRef type, void *pData)
+{
+    _string->Append(Fmt()._clusterPre);
+    Boolean useQuotes = Fmt().QuoteFieldNames();
+    VariantDataRef variantData = *reinterpret_cast<VariantDataRef *>(pData);
+
+    if (useQuotes)
+        _string->Append('\"');
+    _string->AppendCStr(variantInnerData);
+    if (useQuotes)
+        _string->Append('\"');
+    _string->Append(*tsNameSuffix);
+
+    TypeRef underlyingType = variantData->GetInnerType();
```

Comment:
I would rename "underlyingType" to "innerType" to match the API being called.

Reply:
Done.